pegasus/sites.v3/code.org/public/ai/index.haml

```diff
@@ -130,7 +130,18 @@ theme: responsive_full_width
           %a.link-button{href: CDO.studio_url("/courses/self-paced-pl-aiml-2023")}
             =hoc_s(:call_to_action_self_paced_pl)
 
-= view :section_divider_line
+
+%section.bg-neutral-light
+  .wrapper{style: "display: flex; justify-content: space-between; align-items: center; flex-wrap: wrap"}
+    %figure.col-45
+      %img{src: "/images/ai/pl/101/TeachAI.png", alt: "", style: "width: 100%"}
+    .text-wrapper.col-50
+      %h2{style: "margin-top: 0"}
+        =hoc_s(:teach_ai_title)
+      %p
+        =hoc_s(:teach_ai_description)
+      %a.link-button{href: "teachai.org/toolkit"}
```

Comment:
This link needs to be updated to `https://teachai.org/toolkit` and can you add `target: "_blank", rel: "noopener noreferrer"` here too so it opens in a new tab?

Reply:
I also added a new class in [this PR](https://github.com/code-dot-org/code-dot-org/pull/54121) that will put a little external link icon on buttons that take the user away from the `code.org` site. You can add `.has-external-link` after the `.link-button` class.

Here's an example of how it'll look:

<img width="316" alt="Screenshot 2023-10-13 at 12 22 29 PM" src="https://github.com/code-dot-org/code-dot-org/assets/9256643/9b3d7473-5df6-4680-8d19-bef1f06418a3">